openage/convert/main.py

```diff
@@ -82,7 +88,10 @@ def convert_assets(assets, args, srcdir=None):
     if srcdir is None:
         srcdir = acquire_conversion_source_dir()
 
-    testfile = 'data/empires2_x1_p1.dat'
+    if srcdir['AoK HD.exe'].exists():
+        testfile = 'resources/dat/empires2_x1_p1.dat'
```

Comment:
@jfeo I have updated my AoE HD edition today via steam and tried your media conversion fix. It was not working for me because the file 'resources/dat/empires2_x1_p1.dat' was not found. I changed the path to 
```
testfile = 'resources/_common/dat/empires2_x1_p1.dat'
```
and the media conversion was successful.